### DISCUSSION TITLE
Feature: Install Application at runtime

### DISCUSSION THREAD

**Discussion by kumavis** [DustinBrett/daedalOS#214] [Feature Requests]
having fun playing around with daedalOS, thought it would be fun to be able to download and share applications as well

seems like you would primarily need:
- hook to add `extensions` in `components/system/Files/FileEntry/extensions.ts`
- hook to add to the Processes `directory` in `contexts/process/directory.ts`
- some way to preserve to storage

let me know what you think!

**Comment by DustinBrett** [comment]
Thanks! Glad you've had fun with it. Indeed 3rd party app / install support would be ideal. And when done it would ideally link in with as much as possible. I have some plans for this and will move this to discussions as well as continue the technical details in the PR you opened.
